app/controllers/providers/means/capital_disregards/add_details_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Providers
+  module Means
+    module CapitalDisregards
+      class AddDetailsController < ProviderBaseController
+        prefix_step_with :capital_disregards
+        def show
+          @form = AddDetailsForm.new(model: capital_disregard)
```

Comment:
Should this meet the current pattern, that is to chain the module definitions `Providers::Means::CapitalDisregards::AddDetailsForm`?

Reply:
The forms are namespace as such, but rails doesn't _need_ to chain the modules when the controller is in the same namespace.

The other two controllers are 50/50, one does, one doesn't. I think this one sets the pattern! 😁

Reply:
After a discussion today with all of the Civil Apply devs, it was decided to keep this as it is for now.